test/INP_concentration_parametrization.jl

```diff
@@ -0,0 +1,34 @@
+import Test as TT
+
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+import CloudMicrophysics.Parameters as CMP
+import CloudMicrophysics.HetIceNucleation as CMI_het
+
+@info "INP Concentration Paremetrization Test"
+
+function test_INP_concentration(FT)
+
+    # test for the parametrization of ice nucleation based on Frostenberg et al., 2023
+    TT.@testset "Frostenberg" begin
+
+        ip = CMP.Frostenberg2023(FT)
+
+        test_frostenberg(INPC, T) =
```

Comment:
Why do we need this additional wrapper? We could just call the `INP_concentration_frequency` function in line 25?